src/config/WarehouseCrawlDataConfig.ts

```diff
@@ -0,0 +1,41 @@
+const URL = 'https://www.cbre-propertysearch.jp/industrial/';
+
+const LIST_PROVINCE = {
+  DOM_PROVINCE : '#contents > div > div.propertyList > div > div.itemGroup >.item',
+  DOM_WAREHOUSE : 'div.inner > div > div.body > div.head > h2 > a',
+  DOM_LOCATION : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(1) > td',
+  DOM_TRAFFIC : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(2) > td',
+  DOM_SCALE : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(3) > td',
+  DOM_COMPLETION : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(4) > td',
+};
+
+const LIST_STORE = {
+  DOM_STORE: '#contents > div > div.columnSection.clearfix',
+  DOM_IMAGE_1: 'div > div.imgSection > div > div.photo > div.inner > div > div:nth-child(1) > img',
+  DOM_IMAGE_2: 'div > div.imgSection > div > div.photo > div.inner > div > div:nth-child(2) > img',
+  DOM_NAME: 'div > div.bodySection > table > tbody > tr:nth-child(1) > td',
+  DOM_LOCATION: 'div > div.bodySection > table > tbody > tr:nth-child(2) > td',
+  DOM_TRAFFIC: 'div > div.bodySection > table > tbody > tr:nth-child(3) > td',
```

Comment:
có quá nhiều cái `div > div.bodySection > table > tbody`
mà cái này là DOM chỉ đến cái table detail
```suggestion
  DOM_TABLE_DETAIL_INFO: 'div > div.bodySection > table > tbody'
  DOM_TRAFFIC: `${DOM_TABLE_DETAIL_INFO} > tr:nth-child(3) > td`,
```